drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -378,12 +385,12 @@ function DragAndDropTemplates(configuration) {
     var sidebarButtonTemplate = function(buttonClass, iconClass, buttonText, options) {
         options = options || {};
         if (options.spinner) {
-            iconClass = 'fa-spin.fa-spinner';
+            iconClass = 'fa-spin fa-spinner';
```

Comment:
Not sure how we can let the user know the show answer button is doing something if deleted

Reply:
@lcamacho, it would still have text, wouldn't it?

![image](https://user-images.githubusercontent.com/3189670/161313171-31b0a180-1212-45f3-8d11-6af5f801b0d4.png)

Reply:
Ok, now it says `Processing...` instead of showing the spinner

Reply:
@lcamacho, sorry, I've referenced an incorrect part of the code. I've meant the `fa-refresh` icon of the "Reset" button and the `fa-info-circle` icon of the "Show Answer" one. However, it would need some more reworks to look good natively, so I've reverted this change. We'll just change this through a custom theme.